ImplementationGuide/markdown/UebergreifendeFestlegungen/UebergreifendeFestlegungen_Suchparameter.md

```diff
@@ -1,8 +1,14 @@
+---
+topic: UebergreifendeFestlegungen-UebergreifendeFestlegungen-Suchparameter
+---
 ## Allgemeine Hinweise zu Suchparametern
+Originäre ISiK Use Cases sind versorgungsorientiert und patientenorientiert. Dies resultiert darin, dass in der Profilierung der ISiK-Datenobjekte das Vorhandensein einer Referenz auf ISiKPatient (Patient) und ISiKKontaktGesundheitseinrichtung (Encounter) wo möglich gefordert wird. Entsprechend sind Abfragen durch Clients auf Basis von bekannten Informationen aus *einer* Patient- und/oder Encounter-Ressource zu begrenzen (Abfragen auf Patientenkohorten oder sonstige Forschungsabfragen sind nicht im Fokus von ISiK).
```

Comment:
question: "aus" einer Patient- und/oder Encounter-Ressource oder AUF eine Patient- und/oder Encounter-Ressource zu begrenzen?

Reply:
comment: Gute Frage. "Aus" macht hier keinen Sinn. Die Formulierung ist allgemein unglücklich - es ging hier im Wesentliche darum klarzustellen, dass ISIK nicht primär den Use Case von Massenabfragen adressiert, sondern einzelne Patienten und einzelne Fälle. 

FYI @simoneOnFhir - sollten wir ggf. auch noch in Stufe 4 überarbeiten.

Reply:
Hier ist *aus* m.E. korrekt. Ich kenn den Identifier "123452 *aus* der Patienten-Ressource und suche jetzt die Diagnosen dazu:
/Condition?patient.identifier=12345